app/views/groups/index.html.erb

```diff
@@ -49,33 +49,7 @@
     <%= link_to t('upload'), 'javascript:void(0);', onclick:'modal_upload.open();' %>
 <% end %>
 
-<div id="notice">
-  <div class="success" id="add_success" style="display: none;"></div>
-  <% # This is for error display injected by JavaScript %>
-
-  <div class="error" id="errors" style="display: none; ">
```

Comment:
ids here suggest there was some related Javascript. Can you please do a search, and if it can be removed (likely), remove it.